sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/ControlFlowGraph/UcfgExpression.cs

```diff
@@ -184,6 +184,14 @@ public override void ApplyAsTarget(Instruction instruction)
             }
         }
 
+        internal class ElementAccessExpression : MemberAccessExpression
```

Comment:
I would probably still go for `IndexerAccessExpression` but I understand `Element` is what roslyn defines.